modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -140,6 +140,14 @@ if (showSource) {
 name = HtmlUtil.escapeJS(name);
 %>
 
+<aui:script require="frontend-js-web/index as frontendJsWeb">
+	var {getEditorConfigurationCX} = frontendJsWeb;
+
+	if (!Liferay.Util.getEditorConfigurationCX) {
```

Comment:
I know this is a PoC, but be aware that Brian will request you to rename CX to ClientExtension ;-)

Reply:
yep, I wanted you guys to carefully examinate the approach as it's extremely simple and it's been in front of our eyes all this time. 
Naming is not the most salient thing in this PR though...